sw/device/lib/crypto/drivers/aes_test.c

```diff
@@ -9,23 +9,44 @@
 #include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 
-static const char kSecretKey[] = "Secret test key.";
+// NIST 800-38a F.5.1 CTR-AES128.Encrypt test vectors.
+static const uint32_t kSecretKey[] = {
```

Comment:
I wonder if I should convert these into KATs.

Reply:
We'll eventually need to run the larger ACVP test sets for AES, but I think that should be done through the top-level API so we ensure everything is correct at that entrypoint.

Reply:
SG